setup.py

```diff
@@ -43,6 +43,7 @@
         'Acquisition',
         'collective.cover',
         'collective.portlet.calendar',
+        'cssselect',
```

Comment:
shouldn't this be a test dependency?

Reply:
not in this case.. the exception raised when start the instance in my tests.